src/index.html

```diff
@@ -48,20 +48,12 @@ <h1 class="flex">What's Cookin, Good Lookin?</h1>
             <section class="flex column full-width med-brdr-radius blue">
 							<p class="text-center" id="bigModalCost">$40.75</p>
             </section>
-            <div class="flex column around basis half-width med-top-marg">
-							<div class="flex column" id="bigModalHeart">
-								<i class="far fa-heart fa-10x fa-cog-heart clickable"></i>
-							</div>
-							<div class="flex column" id="bigModalPlus">
```

Comment:
Honestly, an excellent solution. Let's simplify first, make more complicated as we grow from a perfectly functioning base